backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -487,12 +488,19 @@ open class SubmissionDatabaseService(
 
         val groupCondition = getGroupCondition(groupIdsFilter, authenticatedUser)
 
+        val submitterCondition = if (submitterNamesFilter !== null) {
```

Comment:
Maybe we should also include empty list here? Depends what we want empty list to do. Right now empty list means nothing gets approved, which is fine but then what's the point of calling the endpoint, though it does make some client code easier potentially. I'm undecided.

Reply:
> Right now empty list means nothing gets approved

I actually like this option so I will leave like this if thats ok